apps/studio/components/interfaces/Organization/InvoicesSettings/InvoicesSettings.tsx

```diff
@@ -61,6 +68,27 @@ const InvoicesSettings = () => {
     )
   }
 
+  if (
+    selectedOrganization?.managed_by !== undefined &&
+    selectedOrganization?.managed_by !== 'supabase'
+  ) {
+    // TODO: get vercel invoices link
+
+    return (
+      <ScaffoldContainerLegacy>
+        <PartnerManagedResource
+          partner={selectedOrganization?.managed_by}
+          resource="Invoices"
+          ctaUrl={
+            selectedOrganization?.managed_by === 'vercel-marketplace'
+              ? `https://vercel.com/${'supabase'}/~/integrations/products/${process.env.NEXT_PUBLIC_VERCEL_MARKETPLACE_INTEGRATION_NAME}/${selectedOrganization.partner_id}/invoices`
```

Comment:
maybe we just need to use our own API here to do the redirect, since we need to call Vercel and get the base url of the team id

Reply:
yep updating this 👍🏻